### ISSUE TITLE
Support for nuget artefacts

### ISSUE THREAD

**Issue by mariotoffia** [eclipse/packagedrone#69]
It would be really complete if packagedrone would support Nuget artefacts. In our organization we are many java and .net developers. (Like https://github.com/JetBrains/teamcity-nuget-support/tree/v0.14/nuget-server)

Cheers,
 Mario

**Comment by ctron** [comment]
I totally agree! I peeked into that a while ago but found it a rather complex interface. So I did not pursue it since I don't have a direct use case for it myself.

But I am happy to help everyone working on it!
